src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1845,11 +1845,10 @@ protected override void InvokeDefaultAction(string path)
                                 //  a) the user has asked to with the -FollowSymLinks switch parameter and
```

Comment:
@SteveL-MSFT, your last commit had 2 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/test/tools/CodeCoverageAutomation/README.md.https://coveralls.io/github/PowerShell/PowerShell should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/testing-guidelines/TestRoadmap.md.https://coveralls.io/github/OpenCover/opencover should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```